### PR TITLE
Update tbcore.js - Added new quotes

### PR DESCRIPTION
**Pull request by Newcool1230**
Added new quotes

### CODE REVIEW

File: extension/data/tbcore.js [hunk 1]
```diff
@@ -98,7 +98,18 @@ function initwrapper ({userDetails, newModSubs, cacheDetails}) {
                   'Releasing raptors',
                   'Booting robot uprising',
                   'I need to tell you something critically important! I am sure I will remember in a moment...',
-                  "/u/dakta ran out for a pack of smokes... BUT HE PROMISED HE'D BE RIGHT BACK"];
+                  "/u/dakta ran out for a pack of smokes... BUT HE PROMISED HE'D BE RIGHT BACK",
+                  'Yeeting your feedback to the trashcan.',
```
Comment (creesch) [inline extension/data/tbcore.js]: Not a fan of this one.

File: extension/data/tbcore.js [hunk 2]
```diff
@@ -98,7 +98,18 @@ function initwrapper ({userDetails, newModSubs, cacheDetails}) {
...
+                  'Here listen to some music while you wait, https://youtu.be/dQw4w9WgXcQ',
+                  'Me? No. I\'m no docter but it looks like you have a broken toe.',
+                  'Boo! Scared yeh didn\'t I?',
+                  'If it looks like a goat, sounds like a goat, acts like a goat. Then its probably not a duck.',
```
Comment (creesch) [inline extension/data/tbcore.js]: Most of the other quotes are an indication of action however silly they are, this one is just a quote ;)

File: extension/data/tbcore.js [hunk 3]
```diff
@@ -98,7 +98,18 @@ function initwrapper ({userDetails, newModSubs, cacheDetails}) {
...
+                  'The last thing you want to do is needing to pee before a battle.',
```
Comment (creesch) [inline extension/data/tbcore.js]: Also not an indication of action. *"Hold on, need a bathroom break!"* or something along those lines would work though.

File: extension/data/tbcore.js [hunk 4]
```diff
@@ -98,7 +98,18 @@ function initwrapper ({userDetails, newModSubs, cacheDetails}) {
...
+                  'When in doubt, check the lost and found.',
+                  'Can you check under the table? I\'ve seem to have lost my pants.',
+                  'Don\'t rustle my jimmies.',
```
Comment (creesch) [inline extension/data/tbcore.js]: Same as the others *"Rustling some jimmies"* would be the appropriate quote here ;)

Comment (Newcool1230) [reply extension/data/tbcore.js]: Rustling some jimmies

**Pull request merged**